src/components/NostoProduct.ts

```diff
@@ -113,3 +113,10 @@ function registerATCButtons(element: NostoProduct, { addToCart, selectSkuId }: S
     })
   })
 }
+
+function registerSkuData(element: NostoProduct, { setSkus }: Store) {
+  const dataEl = element.querySelector("script[n-sku-data]")
```

Comment:
When we introduce options to supply the data this way, should we also enforce typing and validation for data structure? A lightweight validation framework maybe?

Reply:
Or do we expect it to be handled in the client script?

Reply:
I can add basic validation